flexport/latitude
# Updated story imports to be relative. Flow now runs on stories.

This PR updates imports in the `stories/` directory to be relative.
This PR also removes `module.system.node.resolve_dirname=stories` from `.flowconfig`, which causes flow to run on the stories directory.

There are also some one-off fixes that resolve flow errors.

## Manual Test Plan
checkout this branch,
open storybook `yarn storybook`
see that stories still load
